web/impact/impact/v1/views/user_detail_view.py

```diff
@@ -68,3 +74,13 @@ def patch(self, request, pk):
                 profile.save()
         user.save()
         return Response(status=200)
+
+    def partner_organizations(self, user):
+        team = PartnerTeamMember.objects.filter(
+            team_member=user)
+        return team.values_list('partner__organization_id', flat=True)
+
+    def startup_organizations(self, user):
+        team = StartupTeamMember.objects.filter(
```

Comment:
This should use user.startupteammember_set() instead of starting from the class.